l10n_ch_hr_payroll/README.rst

```diff
@@ -1,31 +1,73 @@
-Swizerland Payroll Rules.
-=========================
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
 
-**Features list :**
-    * Add Swiss salary rule categories
-    * Add Swiss salary rules
-    * Add children in school to employee
-    * Add LPP range to contract
+Module name
+===========
 
-**For functionnal information:**
-http://ur1.ca/ir5ou
+This module allows you to manage the salaries of your employees
 
-**Author :**
-Open Net Sàrl -Industrie 59 1030 Bussigny Suisse - http://www.open-net.ch
+For further information, please visit:
 
-**Contact :**
-info@open-net.ch
+* http://ur1.ca/ir5ou
 
 **History :**
 
+Attention: To prevent overwriting your salary rules changes, 
+               an update from 1.0.5 and lower to 1.0.6 and higher 
+               creates duplicates of the salary rules.
+
+Known issues / Roadmap
+======================
+
 V1.0.0: 2014-11-07/Sge
-    * Add Salary rule categories
-    * Add Salary rules
-    * Add Employee children in school
-    * Add Contract LPP rate
+    * Add Salary rule categories.
+    * Add Salary rules.
+    * Add Employee children in school.
+    * Add Contract LPP rate.
 
 V1.0.1: 2014-11-11/Sge
-    * Set the 'LPP rate'' digits to 'Payroll Rate' decimal accuracy
+    * Set the 'LPP rate'' digits to 'Payroll Rate' decimal accuracy.
 
 V1.0.2:
     * Add some minor changes, based on pull request #66 comments.
+
+V1.0.3-4:
+    * Add LPP Amount to contract.
+    * Add Worked Hours Rate to contract.
+    * Add Hourly Rate to contract.
+    * Compute the Wage of contract, based on Worked Hours and Hourly Rate.
+    * Add new salay rules
+
+V1.0.5:
+    * Add Holiday Rate to contract.
+    * Update "Indemnité vacances 8,33%" rule to include 
+      "Holiday Rate" from contract.
+
+V1.0.6:
+    * Move salary rules from CSV file to XML file.
+    * Import salary rules only at install.
+    * Internal reorganization of files (.py and .xml).
+
+Credits
+=======
+
+Contributors
+------------
+
+* Sebastien Legendre <sge@open-net.ch>
+* Yvon-Philippe Crittin <cyp@open-net.ch>
+
+Maintainer
+----------
+
```

Comment:
Here maintainer should be Odoo community association as in template.

We want issues reported on github to track them as we do for every single module included in the OCA.